shared/templates/template_BASH_mount_option_removable_partitions

```diff
@@ -4,19 +4,15 @@
 # Include source function library.
 . /usr/share/scap-security-guide/remediation_functions
 
-populate {{{ MOUNTPOINT }}}
+populate var_removable_partition
 
-include_mount_options_functions
+device_regex="^\s*$var_removable_partition\s\+"
+mount_option="{{{ MOUNTOPTION }}}"
 
-function perform_remediation {
-	# test "$mount_has_to_exist" = 'yes'
-	if test "{{{ MOUNT_HAS_TO_EXIST }}}" = 'yes'; then
-		assert_mount_point_in_fstab "${{{ MOUNTPOINT }}}" || { echo "Not remediating, because there is no record of ${{{ MOUNTPOINT }}} in /etc/fstab" >&2; return 1; }
-	fi
-
-	ensure_mount_option_in_fstab "${{{ MOUNTPOINT }}}" "{{{ MOUNTOPTION }}}" "{{{ FILESYSTEM }}}" "{{{ TYPE }}}"
```

Comment:
I cannot find where `{{{ FILESYSTEM }}}` was set.

Reply:
it wasn't set anywhere